lib/StyleCop/StyleCopCheck.proj

```diff
@@ -0,0 +1,34 @@
+<Project DefaultTargets="Full" xmlns= "http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+
+    <UsingTask AssemblyFile="StyleCop.dll" TaskName="StyleCopTask"/>
+
+    <PropertyGroup>
+        <FolderToAnalyse>..\..\src</FolderToAnalyse>

```

Comment:
@ferventcoder looks like I can set all the MSBuild properties from within the .step file, so we can have some defaults in this .proj file, and override them with specific versions from the .step file.